### PR TITLE
Add notification for powwow

### PR DESCRIPTION
**Pull request by tbille**
# Done 

Add scripts for the powwow crew

# QA

- `dotrun start-irc` (make sure you have credentials)
- `curl --request POST -F "secret=secret_notification" http://localhost:8080/hubot/notify-powwow-form\?room\=#webbot-canonical-web-test`
- should have a cool message in the channel #webbot-canonical-web-test

### CODE REVIEW

**Comment by sowasred2012** [comment]
Moving this back to current iteration - I guess this needs to be updated for Mattermost now?

**Comment by tbille** [comment]
@sowasred2012 this will work with Mattermost :smile:

File: scripts/powwow.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    robot.router.post("/hubot/notify-powwow-form", async function(req, res) {
+        var query = querystring.parse(url.parse(req.url).query);
+        var data = req.body;
+        if (!query.room) {
+            res.send("Parameters rooms required");
+            res.end("");
+        }
+
+        if (data.secret && data.secret === SECRET_KEY) {
+	          robot.messageRoom(query.room, "😎 Hey hey hey, remember to add some questions in the form to get a great Powwow Quiz this Friday: https://forms.gle/j7J2Bbvo7usN7eqX9");
```
Comment (sowasred2012) [inline scripts/powwow.js:35]: I agree with you about having @webteam here would grab a bit more attention

**Comment by amylily1011** [comment]
damn! just saw this

**Review by sowasred2012 — APPROVED** [review]
LGTM!

**Pull request merged**